### ISSUE TITLE
Predefine OEM Drive/Library 

### ISSUE THREAD

**Issue by MrSchism** [ValveSoftware/SteamOS#271]
SteamOS/Steam Linux Client needs an OEM tool for creating predefined Steam library locations.
It'll help with multi-drive installs. 

Example: SteamOS/client installed on 16GB mSATA SSD (/) with library only on 3TB HDD (/library).  A way of controlling it so ONLY the HDD gets the library and the SSD only gets the OS/Client would be insanely useful.  Perhaps have Steam read from a file on first run or have a file designate library locations for Steam in general.

**Comment by not83** [comment]
Yeah that'll be an absolute must. Let's hope a dev sees this request
